utils/common/src/main/kotlin/Extensions.kt

```diff
@@ -210,29 +207,6 @@ inline val Int.gibibytes get(): Long = mebibytes * 1024L
  */
 fun <T> Iterator<T>.nextOrNull() = if (hasNext()) next() else null
 
-/*
```

Comment:
commit-msg: Should this change be marked bracing?

Reply:
While I was trying to avoid major version bumps, formally you are correct, even if these are utilities mostly mean for ORT internal use.

So I went ahead and consistently marked all commit breaking that would require code changes from any arbitrary user of those libraries.